### PR TITLE
Add aliyun cloud support

### PR DESCRIPTION
**Pull request by jwang11**
Enable ucd to work in aliyun cloud by add a config templete
for aliyun

Signed-off-by: jwang <jing.j.wang@intel.com>

### CODE REVIEW

**Comment by jwang11** [comment]
@ahkok any comments?

File: src/ucd-data-fetch.c [hunk 1]
```diff
@@ -103,6 +103,18 @@ static struct cloud_struct config[MAX_CONFIGS] = {
 		"    groups: wheelnopw\n" \
 		"ssh_authorized_keys:\n" \
 		"  - "
+	},
+	{
+		"aliyun",
+		"100.100.100.200",
```
Comment (ahkok) [inline src/ucd-data-fetch.c]: This is not a link-local address, again. Same problem as Tencent. Not OK.

**Comment by ahkok** [comment]
```
$ make distcheck
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/ahkok/clr/projects/micro-config-drive/missing aclocal-1.16 -I m4
 cd . && /bin/sh /home/ahkok/clr/projects/micro-config-drive/missing automake-1.16 --foreign
configure.ac:8: error: required file 'data/ucd-aliyun.service.in' not found
make: *** [Makefile:551: Makefile.in] Error 1
```

You need to create `data/ucd-aliyun.service`

**Comment by ahkok** [comment]
4706d35c2c9f7d77ccbf67109659a69982928679

**Pull request closed without merging**